Sources/Build/BuildOperation.swift

```diff
@@ -526,9 +526,15 @@ public final class BuildOperation: PackageStructureDelegate, SPMBuildCore.BuildS
             // FIXME: This is super unfortunate that we might need to load the package graph.
             let graph = try getPackageGraph()
 
+            let buildTriple: BuildTriple? = if let destination {
+                    destination == .host ? .tools : .destination
```

Comment:
Is this how `swiftformat` actually indents it? 😦

Reply:
Looks like I might have ran the wrong one again...

Reply:
Fixed, thanks!